chromium_src/chrome/common/url_constants.cc

```diff
@@ -30,14 +30,14 @@ const char kCastNoDestinationFoundURL[] =
     "https://support.brave.com/";
 
 const char kChooserBluetoothOverviewURL[] =
-    "https://support.brave.com/";
+    "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
 
-const char kChooserHidOverviewUrl[] = "https://support.brave.com/";
+const char kChooserHidOverviewUrl[] = "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
 
-const char kChooserSerialOverviewUrl[] = "https://support.brave.com/";
+const char kChooserSerialOverviewUrl[] = "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
```

Comment:
Chromium C++ code style follows a Google one and there is a [line length rule](https://google.github.io/styleguide/cppguide.html#Line_Length) we should abide.
Please run `clang-format` on this lines.

Reply:
@goodov thanks for informing me about the style guide. I've formatted the file according to it.